procedures/unit-testing-tracing.ipf

```diff
@@ -988,4 +999,88 @@ static Function AnalyzeTracingResult()
 	UTF_Reporting#UTF_PrintStatusMessage(statOut)
 End
 
+Function IUTF_RestoreTracing()
+	variable i, size
+	string path, backupPath, msg
+
+	WAVE/T wvProcs = GetTracedProcedureNames()
+	size = DimSize(wvProcs, UTF_ROW)
+
+	if(size == 1 && !strlen(wvProcs[0]))
+		UTF_Reporting#UTF_PrintStatusMessage("Nothing to restore")
+		return NaN
+	endif
+
+	sprintf msg, "%d procedure files to restore", size
+	UTF_Reporting#UTF_PrintStatusMessage(msg)
+
+	for(i = 0; i < size; i += 1)
+		path = GetProcedurePath(wvProcs[i])
+		backupPath = path + ".backup"
+
+		GetFileFolderInfo/Q/Z=1 backupPath
+		if(V_flag)
+			sprintf msg, "Backup file not found: %s", backupPath
+			UTF_Reporting#UTF_PrintStatusMessage(msg)
+			continue
+		else
+			sprintf msg, "Backup restored for %s", path
+			UTF_Reporting#UTF_PrintStatusMessage(msg)
+		endif
+
+		MoveFile/O/Z=1 backupPath as path
+		if(V_flag)
+			sprintf msg, "Cannot move %s to %s (error: %d)", S_fileName, S_path, V_flag
+		endif
+	endfor
+
+	WriteProcList("", includeTag = 0)
+
+	UTF_Reporting#UTF_PrintStatusMessage("Restoring procedure files from backup completed.")
+	CompileAndRestart()
+End
+
+static Function/S GetProcedurePath(string procWin)
+	string allProcWins, macrosList, funcList, path, name, msg
+	variable i, size, err
+
+	allProcWins = UTF_Basics#GetProcedureList()
+	if(WhichListItem(procWin, allProcWins) == -1)
+		sprintf msg, "Procedure window %s not found.", procWin
+		UTF_Reporting#ReportErrorAndAbort(msg)
+	endif
+
+	macrosList = MacroList("*", ";", "KIND:7,WIN:" + procWin)
+	funcList = FunctionList("*", ";", "KIND:18,WIN:" + procWin)
+
+	WAVE/T wMacroList = ListToTextWave(macrosList, ";")
+	size = DimSize(wMacroList, UTF_ROW)
+	for(i = 0; i < size; i += 1)
+		path = MacroPath(wMacroList[i])
+		if(!UTF_Utils#isEmpty(path))
+			return path
+		endif
+	endfor
+
+	WAVE/T wFuncList = ListToTextWave(funcList, ";")
+	size = DimSize(wFuncList, UTF_ROW)
+	for(i = 0; i < size; i += 1)
```

Comment:
`for(funcName : wFuncList)`

Reply:
After reordering the commit this function does no longer exists.